docs/bloodmoon-cult/bloodmoon-cult.mdx

```diff
@@ -1,7 +1,6 @@
 ---
-name: Bloodmoon Cult
-menu: The Bloodmoon Cult
```

Comment:
This is now determined by folder structure and the `_category_.json` file to give it a nicer name.